packages/wrapped-keys/src/lib/litActions/solana/src/signMessageWithSolanaEncryptedKey.js

```diff
@@ -0,0 +1,70 @@
+const { Keypair } = require('@solana/web3.js');
+const bs58 = require('bs58');
+const nacl = require('tweetnacl');
```

Comment:
Where have these been injected? It needs to be bundled in the Lit Action code before publishing to IPFS

Reply:
When you install the package there's a `postinstall` NPM script that executes the bundler (can also be called manually via `yarn bundle` within the `wrapped-keys` package). Afterwards, the bundled version of these files are located in `packages/wrapped-keys/src/lib/litActions/solana/dist` and are currently manually uploaded to IPFS

I think we should create a script to publish to IPFS, but I figured that could come after this PR is merged

Reply:
Yeah we can write the IPFS uploading script later, out of scope for this PR

Reply:
> postinstall NPM script that executes the bundle

Which packages are being bundled? We should only bundle the necessary ones as we're uploading it to IPFS so the file size should be as small as possible

Reply:
```ts
const { Keypair } = require('@solana/web3.js');
const bs58 = require('bs58');
const nacl = require('tweetnacl');
```

@DashKash54  These imports are bundled and are being used by the LA. Unminified, the bundle is 1 MB, bundled it's about 400 KB

Reply:
We can restrict the required function from nacl